pkg/controller/util/federated_informer.go

```diff
@@ -235,7 +235,7 @@ func NewFederatedInformer(
 						}
 					}
 				} else {
-					glog.V(4).Infof("Cluster %v not updated to %v as ready status and specs are identical", oldCluster, curCluster)
+					glog.V(5).Infof("Cluster %v not updated to %v as ready status and specs are identical", oldCluster, curCluster)
```

Comment:
Maybe push to 7 - relegate this kind of "I'm doing my job" low-value log to the bottom of the barrel.

Reply:
Haha, I had it at 6 originally and then lowered it in case somebody would object. :)